components/AppBar/AppBar.js

```diff
@@ -1,7 +1,7 @@
 import Toolbar from '@material-ui/core/Toolbar'
 import * as Styles from './AppBar.styles'
 import IconButton from '@material-ui/core/IconButton'
-import LogoutButton from '../buttons/LogoutButton'
+import Typography from '@material-ui/core/Typography'
 
 export default function AppBar({ props }) {
 	const { loggedIn, toggleDrawer } = props
```

Comment:
Just destructure these from passed props to reduce the logic trail 
```suggestion
export default function AppBar({ loggedIn, toggleDrawer }) {
```